### PR TITLE
TOOLS-2845: add changelog for 100.4.0

### PR DESCRIPTION
**Pull request by huan-Mongo**

### CODE REVIEW

**Review by tfogo — APPROVED** [review]
Looks good, just a small nit. I think it's good to go once we fix that.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,90 @@
...
+</li>
+</ul>
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
+<h3>        Bug
+</h3>
+<ul>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2041'>TOOLS-2041</a>] -         Mongorestore should handle duplicate key errors during oplog replay
+</li>
+<li>[<a href='https://jira.mongodb.org/browse/TOOLS-2833'>TOOLS-2833</a>] -         creating and index with partialFilterExpression during oplogReplay fails
```
Comment (tfogo) [inline CHANGELOG.md:38]: Small nit, I know the ticket title is incorrect but can we change this to "Creating an"?

**Review by tfogo — APPROVED** [review]
Didn't submit the comment before submitting the review... here's the nit:

File: CHANGELOG.md (hunk 1 above)
Comment (huan-Mongo) [reply CHANGELOG.md:38]: Sure!

**Pull request merged**